src/jsLib.gs

```diff
@@ -343,6 +349,18 @@ function convertArrayToObj_(arrayInput, keyFunction) {
   return objectToReturn;
 }
 
+/**
+ * @TODO: Move to appropiate file (not jiraCommand.gs, jsLib.gs, but where?) Get a sheet from current active Spreadsheet by ID passed.
```

Comment:
models/gas/GoogleSheet.gs ?

Reply:
Its not a model! And i really dont like breaking apart so much code in so many files. This method per se is more fitting to a "utils" like collection/file.

Reply:
yup makes sense. 
I think we need a folder for stuff that is directly related to Google Sheets UI code. /view/utils/???